game_eggs/steamcmd_servers/palworld/egg-palworld.json

```diff
@@ -0,0 +1,96 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-01-20T14:16:27+01:00",
+    "name": "Palworld",
+    "author": "admin@ballaual.de",
+    "description": "Fight, farm, build and work alongside mysterious creatures called \"Pals\" in this completely new multiplayer, open world survival and crafting game!",
+    "features": [
+        "steam_disk_space"
+    ],
+    "docker_images": {
+        "ghcr.io\/parkervcp\/steamcmd:debian": "ghcr.io\/parkervcp\/steamcmd:debian"
+    },
+    "file_denylist": [
```

Comment:
delete the file insteed of add it to denylist. i think, this "list" dont work

Reply:
a simple:

rm -f PalServer.sh

is easier ;)

Reply:
doesnt get it restored with the next upate anyways?